wisembly/amqp-bundle/1.3/Makefile

```diff
@@ -0,0 +1,16 @@
+CONSOLE=bin/console
+
+ifndef GATE
+gate:
+	$(error GATE not set)
+endif
+
+gate:
+
+consume: gate $(CONSOLE)
+	$(CONSOLE) wisembly:amqp:consume $(GATE) $(OPTIONS)
+
+publish: gate $(CONSOLE)
+	$(CONSOLE) wisembly:amqp:publish $(GATE) "$(MESSAGE)"
```

Comment:
why making people define env variables when calling the command instead of using arguments ?

Btw, if arguments are used, I don't think wrapping the Symfony command in the makefile makes much sense, as they would just be proxied directly